bin/runtime/src/lib.rs

```diff
@@ -859,59 +859,59 @@ impl pallet_proxy::Config for Runtime {
 #[cfg(not(feature = "liminal"))]
 construct_runtime!(
     pub struct Runtime {
-        System: frame_system,
-        RandomnessCollectiveFlip: pallet_insecure_randomness_collective_flip,
-        Scheduler: pallet_scheduler,
-        Aura: pallet_aura,
-        Timestamp: pallet_timestamp,
-        Balances: pallet_balances,
-        TransactionPayment: pallet_transaction_payment,
-        Authorship: pallet_authorship,
-        Staking: pallet_staking,
-        History: pallet_session::historical,
-        Session: pallet_session,
-        Aleph: pallet_aleph,
-        Elections: pallet_elections,
-        Treasury: pallet_treasury,
-        Vesting: pallet_vesting,
-        Utility: pallet_utility,
-        Multisig: pallet_multisig,
-        Sudo: pallet_sudo,
-        Contracts: pallet_contracts,
-        NominationPools: pallet_nomination_pools,
-        Identity: pallet_identity,
-        CommitteeManagement: pallet_committee_management,
-        Proxy: pallet_proxy,
+        System: frame_system = 1,
```

Comment:
I think it would be great to run subwasm on this change. I suspect it's 0-based by default :P

Reply:
@kostekIV have you run subxt for standard runtime as well? if so (no changes), then it might be 1-based indeed...

Reply:
nope, of course its from 0. Fixed